app/views/participant_signup/_contact.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= f.inputs :name => "Contact Information" do -%>
```

Comment:
I was working with Aayush as these partials were being developed, and the logic of modifying hint and partial content based on context looked ugly enough to where separate partials made sense at the time.

But after looking at the changes you made for supporting parent information (that all looks great by the way!) if we continued with this split, you would have needed to update two separate views which feels even uglier.

My biggest question is if there's any way to better to select form label content than having a ternary or `unless` statement on every field in the partial. The `content_for` method or unorthodox use of the `i18n` package come to mind, but I really don't have much experience here.